server/public/styles.css

```diff
@@ -155,6 +165,36 @@ div > h3 {
   animation-iteration-count: infinite;
 }
 
+.big-container,
+.home {
+  font-weight: bolder;
+  font-family: sans-serif;
+  background-color: lightcyan;
+}
+
+.selected-hotel {
+  width: 200px;
+}
+
+.gradient {
+  background-image: linear-gradient(#f8a170, #ffcd61);
+}
+
+.bottom-nav-abs {
+  position: fixed;
+  bottom: 0 !important;
+  width: 100%;
```

Comment:
Bootstrap has sizing helpers.

https://getbootstrap.com/docs/4.5/utilities/sizing/#relative-to-the-parent